packages/editor/src/components/document-outline/item.js

```diff
@@ -29,7 +29,8 @@ const TableOfContentsItem = ( {
 			}
 		) }
 	>
-		<button
+		<a
+			href={ '#' + level }
```

Comment:
This is not a valid navigable link? Shouldn't it be going to the unique `id` of the element to which it should navigate, not just the node name corresponding to the heading? Otherwise we can have multiple identical `href` which navigate to different headings.

![image](https://user-images.githubusercontent.com/1779930/47532471-7a0dbe00-d87e-11e8-9cbb-54aed79947bc.png)

Put another way: Is there really any benefit to this pull request if we're not also able to drop the `onClick` handler?

Reply:
> Shouldn't it be going to the unique id of the element to which it should navigate

Probably, I was trying to keep my changes to a minimum - i'll take a look.

Reply:
this is fixed